webserver/views/api/v1/dataset_eval.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import absolute_import
+from flask import Blueprint, jsonify, request
+from flask_login import current_user
+from webserver.decorators import auth_required
+import db.dataset_eval
+
+bp_dataset_eval = Blueprint('api_v1_dataset_eval', __name__)
+
+@bp_dataset_eval.route("/pending-jobs-for-user", methods=["GET"])
+@auth_required
+def pending_jobs_for_user():
+    """Return a list of pending jobs for a user to be evaluated remotely"""
+    jobs = db.dataset_eval.get_pending_jobs_for_user(current_user.id)
+    jobs_list = []
+    for job in jobs:
+        jobs_list.append(job["id"])
+    return jsonify(
+        success=True,
+        jobs=jobs_list
```

Comment:
@gentlecat This is a good point. This is specifically for jobs which have been marked as being evaluated on a remote server, so if we used it for the website as well we would need parameters to be able to select all, local, or remote jobs.
My preference is to make the endpoint do a specific thing but make its name clear that this is its task.
What are the requirements that we have on the website? For now is it just to show all jobs that have been submitted?